src/mbgl/shaders/preludes.cpp

```diff
@@ -24,25 +24,6 @@ precision highp float;
 
 #endif
 
-float evaluate_zoom_function_1(const vec4 values, const float t) {
```

Comment:
I meant to include it just as part of bringing shaders back into sync as much as possible, but it's not tied to either of #9345 or #9456 so I'll take it out.